app/views/plantings/_planting_progress.html.haml

```diff
@@ -0,0 +1,18 @@
+- if DateTime.now.to_date < planting.planted_at
+	= "Progress: 0% - not planted yet"
+	.progress
```

Comment:
I don't like the repetition here - could you factor out the progress bar into another partial? [Here's an example of how to pass variables when rendering partials](https://github.com/Growstuff/growstuff/blob/dev/app/views/alternate_names/show.html.haml#L3).